build.gradle.kts

```diff
@@ -8,10 +8,12 @@ plugins {
     id("java")
     // Kotlin support
     id("org.jetbrains.kotlin.jvm") version "1.5.30"
-    // gradle-intellij-plugin - read more: https://github.com/JetBrains/gradle-intellij-plugin
+    // Gradle IntelliJ Plugin
```

Comment:
let's keep the homepages for easier reference/version check

Reply:
Full URLs are available below with the relevant configuration sections